experimental/list_formatter/Cargo.toml

```diff
@@ -44,4 +44,4 @@ std = ["icu_provider/std", "icu_locid/std"]
 default = ["provider_serde"]
 bench = []
 provider_serde = ["serde", "zerovec/serde", "deduplicating_array"]
-provider_transform_internals = ["std"]
+icu4x_no_human_readable_de = []
```

Comment:
Issue: Features in Rust should be additive, not subtractive.  This is because crates are intended to be compiled only once.  For example, if one crate enables `icu4x_no_human_readable_de`, but another crate actually needs to use JSON, the other crate has no way to disable that feature again.

You likely want to make `icu4x_human_readable_de` be enabled by default.